src/main/java/domain/LottoNumbers.java

```diff
@@ -0,0 +1,38 @@
+package domain;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class LottoNumbers {
+    public static final int SIZE = 6;
```

Comment:
이전 리뷰랑 비슷해요!
`LottoNumbers`의 상수를 전역으로 사용하지 않도록 해볼까요..?

불가피하게 `LottoNumberGenerator`에서는 써야한다면 default 범위로 변경할 수 있어요.
```suggestion
    static final int SIZE = 6;
```

아니면, private으로 할 수 있는 방법은 없을까요?!
> hint) `Lotto`랑 `LottoNumbers`와의 차이는 무엇인지,
'로또는 6개 숫자로 이루어져 있다'의 게임 규칙은 누구의 책임인지 고민해보시면 좋을 것 같아요!
두 객체를 분리한 이유가 궁금해요

Reply:
이번에 객체를 바깥 쪽에서 안쪽(Lotto-> LottoNumbers -> LottoNumber)으로 들어가기 보다 안쪽에서 바깥쪽(LottoNumber -> LottoNumbers -> Lotto)으로 계획 하다 보니 너무 구체적으로 만들어 진거 같습니다. 이러한 방식으로 가니 너무 세세히 객체화 하려는 느낌이 강하다는 것을 나중에 느끼게 되었습니다.
보통 객체에 대해서 생각하실때 가장 바깥쪽 외관부터 생각하시는지 궁금합니다!